osf/utils/migrations.py

```diff
@@ -2,13 +2,32 @@
 import json
 import logging
 
+from contextlib import contextmanager
+
 from website import settings
 from osf.models import NodeLicense, MetaSchema
 from website.project.metadata.schemas import OSF_META_SCHEMAS
 
 logger = logging.getLogger(__file__)
 
 
+@contextmanager
+def disable_auto_now_fields(model):
+    """
+    Context manager to disable updates of all auto_now fields for a given model.
+
+    """
+    for field in model._meta.get_fields():
+        if hasattr(field, 'auto_now') and field.auto_now:
```

Comment:
Similar to above: I could also disable `auto_now_add` fields here, but wasn't sure.